sandbox/modules/identity/src/internalClusterTest/java/org/opensearch/identity/HttpSmokeTestCaseWithIdentity.java

```diff
@@ -88,4 +96,47 @@ protected boolean ignoreExternalCluster() {
         return true;
     }
 
+    protected void startNodes() throws Exception {
+        final String clusterManagerNode = internalCluster().startClusterManagerOnlyNode();
+
+        ClusterStateResponse clusterStateResponse = client(clusterManagerNode).admin()
+            .cluster()
+            .prepareState()
+            .setClusterManagerNodeTimeout("1s")
+            .clear()
+            .setNodes(true)
+            .get();
+        assertNotNull(clusterStateResponse.getState().nodes().getClusterManagerNodeId());
+
+        // start another node
+        final String dataNode = internalCluster().startDataOnlyNode();
+        clusterStateResponse = client(dataNode).admin()
+            .cluster()
+            .prepareState()
+            .setClusterManagerNodeTimeout("1s")
+            .clear()
+            .setNodes(true)
+            .setLocal(true)
+            .get();
+        assertNotNull(clusterStateResponse.getState().nodes().getClusterManagerNodeId());
+        // wait for the cluster to form
+        assertNoTimeout(client().admin().cluster().prepareHealth().setWaitForNodes(Integer.toString(2)).get());
+        List<NodeInfo> nodeInfos = client().admin().cluster().prepareNodesInfo().get().getNodes();
+        assertEquals(2, nodeInfos.size());
+
+        Thread.sleep(1000);
+    }
+
+    protected void ensureIdentityIndexIsGreen() {
+        ClusterHealthResponse clusterHealthResponse = client().admin().cluster().prepareHealth().setClusterManagerNodeTimeout("1s").get();
+
+        assertTrue(
+            ConfigConstants.IDENTITY_DEFAULT_CONFIG_INDEX + " index exists",
```

Comment:
For my own understanding, is there what is specific to the Identity changes being present? Just that the configuration should now have the default identity values? 

If this is based on a similar test that already exists could we not just add this line to check for the identity configuration settings into that test? That is assuming this is based on a similar test that runs the smoke test without Identity and that the identity features will always be included in the tests unless specified explicitly. Maybe I am misunderstanding how the tests are organized however--I know you are more familiar with the structure than I am.

Reply:
There are a few test cases that need to verify that identity is initialized on startup and this method can be re-used to assert that the Identity module has been initialized and the identity system index exists.

Reply:
Sounds good!